stripe/src/main/java/com/stripe/android/payments/StripeBrowserLauncherViewModel.kt

```diff
@@ -17,4 +28,41 @@ internal class StripeBrowserLauncherViewModel : ViewModel() {
             ).toBundle()
         )
     }
+
+    fun logAnalytics(
+        shouldUseCustomTabs: Boolean
+    ) {
+        analyticsRequestExecutor.executeAsync(
+            analyticsRequestFactory.create(
+                analyticsDataFactory.createParams(
```

Comment:
It looks like this code is duplicated in a few places, I wonder if we can pull this logic out into a helper class.  This will also have the benefit of simplifying the viewModel

Reply:
It almost feels like the type of object that could be just injected as it only has the publishable key and application context.

Reply:
I actually have a follow-up PR that simplifies analytics requests across the SDK.